checks.d/mysql.py

```diff
@@ -330,9 +330,10 @@ def _get_config(self, instance):
         options = instance.get('options', {})
         queries = instance.get('queries', [])
         ssl = instance.get('ssl', {})
+        connect_timeout = instance.get('connect_timeout', None)
```

Comment:
If this is a question of what the default setting _should_ be:
- Setting it to `None` preserves the original check behaviour (leaving it to system socket defaults).
- Explicitly setting it to `1 second` changes the current behaviour, and I'm not sure that assuming 1 second would be appropriate for everyone.

The [actual code](https://github.com/PyMySQL/PyMySQL/blob/pymysql-0.6.6/pymysql/connections.py#L815) that changes the [socket timeout](https://docs.python.org/2/library/socket.html#socket.socket.settimeout) accepts any value types `integer/float` as well as `None`.